lib/puppet-lint/plugins/check_hiera.rb

```diff
@@ -49,6 +49,10 @@
     # metrics
     'govuk_logging::logstream::metrics_only',
 
+    # Share the array of asset_manager_routes between assets-origin and static
+    # nginx virtual host configuration
+    'router::assets_origin::asset_manager_routes',
```

Comment:
Minor: This comment suggests that it's this line of code which is doing the sharing, but if I understand correctly, it's merely *allowing* the sharing. It would be good to update the comment accordingly.

Reply:
Also did you consider extracting `asset_manager_routes` as a top-level heira variable so that you didn't need this? It feels a bit of a shame to have to add this exception to the lint-ing.

Reply:
I've updated the comment to:

> # Allow sharing of asset_manager_routes array between assets-origin and static
> # nginx virtual host configuration

Reply:
> Also did you consider extracting asset_manager_routes as a top-level heira variable so that you didn't need this? It feels a bit of a shame to have to add this exception to the lint-ing.

I don't think that would've avoided the problem, unfortunately. We'd still need to be able to lookup that top-level hiera variable from within assets-origin and static config files, and that lookup would require an exception to be added to this file.